test/integration/sonobuoy_integration_test.go

```diff
@@ -32,6 +34,42 @@ func findSonobuoyCLI() (string, error) {
 	return sonobuoyPath, nil
 }
 
+func cleanup(t *testing.T, namespace string) {
+	var stdout, stderr bytes.Buffer
+	deleteCommand := exec.Command(sonobuoy, "delete", "--wait", "-n", namespace)
+	deleteCommand.Stdout = &stdout
+	deleteCommand.Stderr = &stderr
+
+	t.Logf("Cleaning up. Running %q\n", deleteCommand.String())
+	if err := deleteCommand.Run(); err != nil {
+		t.Errorf("Sonobuoy exited with an error: %v", err)
+		t.Log(stdout.String())
+		t.Log(stderr.String())
+	}
+}
+
+// TestSimpleRun runs a simple plugin to check that it runs successfully
+func TestSimpleRun(t *testing.T) {
+	var stdout, stderr bytes.Buffer
+
+	ns := randomNamespace()
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	defer cleanup(t, ns)
+
+	args := fmt.Sprintf("run --image-pull-policy IfNotPresent --wait -p testImage/yaml/job-junit-passing-singlefile.yaml -n %v", ns)
+	runCommand := exec.CommandContext(ctx, sonobuoy, strings.Fields(args)...)
+	runCommand.Stdout = &stdout
+	runCommand.Stderr = &stderr
+
+	t.Logf("Running %q\n", runCommand.String())
+	if err := runCommand.Run(); err != nil {
+		t.Errorf("Sonobuoy exited with an error: %v", err)
```

Comment:
I guess for our first test, just seeing that it ran and exited 0 is OK. Feels a bit empty but it is just a starting place.

Reply:
Sorry, I should have followed up on this PR with a comment. Yes, this is a bit of dull and uninteresting test that doesn't tell us anything more than what we currently have in our CI scripts. I think I just got a bit frustrated with trying to get things working and decided that once it _was_ working, I should just try to get that wrapped up and merged. Admittedly, it was fairly straightforward in the end, I was just frustrated with some silly mistakes I made :upside_down_face: